app/services/widgets/widgets-config.ts

```diff
@@ -1,16 +1,12 @@
 import { AnchorPoint } from '../../util/ScalableRectangle';
 import { $t } from '../i18n';
 import { TAlertType } from './alerts-config';
+import { WidgetType } from './widgets-data';
 
-export type TWidgetType = 'AlertBox' | 'ViewerCount';
+export type TWidgetType = WidgetType.AlertBox | WidgetType.ViewerCount | WidgetType.GameWidget;
```

Comment:
What are benefits of using enums instead strings here? I think since Typescript introduced strict string types we don't need enums in most cases. Strings are easier for debugging

Reply:
it's not so much a benefit, but rather a necessity of using `widgets-config` in place of things like `WidgetDefinitions` where we already have a ton of logic built around our enumerable types. i think eventually when we port all of our widget to React this could be worth exploring well but i don't think it's feasible whilst we have both React and Vue widgets.

Reply:
I think I already have some of the code that relies on the string types. 
Should it help if you just add an `id` field here?

```
GameWidget: { 
 id: WidgetType.GameWidget
 ....
```

So we can keep a new simplified style for the new React widgets and have the backward capability with old Vue code.
Or did I miss something?

Reply:
yea, what i mean is there are several places in the sources/widgets services that don't have access to the string type of the widget but do have access to the enum